<launch>
 <arg name="device_name"                default="/dev/ttyUSB0"/>
 <arg name="baud_rate"                  default="1000000"/>

 <arg name="scan_range"                 default="10"/>

 <arg name="profile_velocity"           default="200"/>
 <arg name="profile_acceleration"       default="50"/>

 <param name="device_name"              value="$(arg device_name)"/>
 <param name="baud_rate"                value="$(arg baud_rate)"/>

 <param name="scan_range"               value="$(arg scan_range)"/>

 <param name="profile_velocity"         value="$(arg profile_velocity)"/>
 <param name="profile_acceleration"     value="$(arg profile_acceleration)"/>

 <node name="position_control" pkg="dynamixel_workbench_controllers" type="position_control" required="true" output="screen"/>

 <node name="urg_node" pkg="urg_node" type="urg_node" required="true" output="screen">
 	<param name="ip_address" value="192.168.0.11"/>
 	<param name="ip_port" value="10940" />
 </node> 

 <node name="laser_trasform_manager" pkg="nodding_laser" type="transform_manager_node" required="true"/>

 </launch>
